app/controllers/orga/mailings_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Orga::MailingsController < Orga::BaseController
+  before_action :normalize_params, only: [:create, :update]
+  before_action :find_mailing, only: [:show, :edit, :update, :destroy]
+
+  def index
+    authorize! :read, :mailing
+    @mailings = Mailing.order('id DESC').page(params[:page])
+  end
+
+  def show
+    authorize! :read, :mailing
```

Comment:
Has eventually the same outcome, but for making it more future-proof: this should probably be
```ruby
authorize!, :read, @mailin
```